README.md

```diff
@@ -83,6 +83,26 @@ ipld.util.deserialize(serialize, (err, node) => {
 })
 ```
 
+## Links
+
+You can link to other IPLD data by using values of an object with a single "/" attribute set to the CID string.
+
+```JavaScript
+const dagCBOR = require('ipld-dag-cbor')
+
+const test = {
+  foo: {
+    '/': 'QmRgutAxd8t7oGkSm4wmeuByG6M51wcTso6cubDdQtuEfL' // CID of dag-pb node
+  }
+}
+
+dagCBOR.util.serialize(test, (err, serialized) => {
+  dagCBOR.resolver.isLink(serialized, 'foo', (err, obj) => {
```

Comment:
Please note that `isLink()` will be removed (https://github.com/ipld/interface-ipld-format/issues/31). But at the moment it's still there.